python/grass/jupyter/interactivemap.py

```diff
@@ -309,36 +309,40 @@ def _import_ipyleaflet(error):
         self.region = None
 
         if crs:
-            if isinstance(crs, str):
-                crs_n = CRS.from_string(crs)
-                crs_dict = {"name": crs_n.to_epsg(), "custom": False}
-            else:
-                crs_dict = crs
-
+            proj = gs.read_command("g.proj", flags="jf", srid=crs)
```

Comment:
I meant `g.proj -g` to get the srid key from the output which contains the EPSG:
```
g.proj -g
name=Lambert Conformal Conic
proj=lcc
datum=nad83
a=6378137.0
es=0.006694380022900787
lat_1=36.16666666666666
lat_2=34.33333333333334
lat_0=33.75
lon_0=-79
x_0=609601.22
y_0=0
no_defs=defined
srid=EPSG:3358
unit=Meter
units=Meters
meters=1
```
You can use parse_command